scripts/i18n/sync_world_map_langs.py

```
@@ -140,16 +136,22 @@ def build_official(langs):
+                    {
+                        "id": s_zh["id"],
+                        "zh": s_zh["name"].strip(),
+                        "en": s["en"],
+                        "ja": s["ja"],
+                        "ko": s["ko"],
+                    }
+                )
+            level_list.append(
+                {
+                    "id": lv_zh["id"],
+                    "name": {l: lv[l]["name"] for l in langs},
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)